e2e/tests/integration/arrival-api-with-exports.ts

```diff
@@ -113,5 +128,70 @@ describe('Arrivals CSV Export', () => {
         })
       })
   });
+
+
+    it('No Export flights according when clicked passenger filters for users with api:view permission', () => {
```

Comment:
it("should not export non passenger flights when the passenger filter is selected",...